react/src/pages/Navigation.tsx

```diff
@@ -147,15 +203,21 @@ export default function Navigation({ username, signout, lastLoginTime }: Navigat
                         >
                             <MenuIcon />
                         </IconButton>
-                        <Typography
-                            component="h1"
-                            variant="h6"
-                            color="inherit"
-                            noWrap
-                            className={classes.title}
-                        >
-                            {pageName}
-                        </Typography>
+                        <Switch>
+                            {routes.map(route => (
+                                <Route path={route.path} exact={route.exact}>
```

Comment:
Console complains about missing `key` prop, even though this map is for a static list. You should be able to use the index like you do with ListItemRouterLink.